solhint.js

```diff
@@ -128,13 +132,30 @@ const readConfig = _.memoize(() => {
   let config = {}
 
   try {
-    const configStr = fs.readFileSync('.solhint.json').toString()
+    let configStr
+
+    if (program.configPath && !fs.existsSync(program.configPath)) {
+      throw new FileNotExistsError('The config file doesnt exist')
+    }
+
+    if (program.configPath && fs.existsSync(program.configPath)) {
+      configStr = fs.readFileSync(program.configPath).toString()
+    }
+
+    if (!program.configPath) {
+      configStr = fs.readFileSync('.solhint.json').toString()
+    }
+
     config = JSON.parse(configStr)
   } catch (e) {
     if (e instanceof SyntaxError) {
       console.log('ERROR: Configuration file [.solhint.json] is not a valid JSON!\n')
       process.exit(0)
     }
+    if (e instanceof FileNotExistsError) {
+      console.log('ERROR: Configuration file [' + program.configPath + '] doesnt exist!\n')
+      process.exit(0)
```

Comment:
This shouldn't exit with a success code.

Reply:
Solved!